core/assets/bundles/bundle_vi.properties

```diff
@@ -1281,8 +1281,8 @@ block.power-void.name = Hủy năng lượng
 block.power-source.name = Nguồn năng lượng
 block.unloader.name = Điểm dỡ hàng
 block.vault.name = Nhà kho
-block.wave.name = Wave
-block.tsunami.name = Tsunami
+block.wave.name = Thủy Chiều
+block.tsunami.name = Sóng Thần
```

Comment:
At lest you dont need to translate turret name.

Reply:
~~skill issue~~